diff --git a/wscript b/wscript
index 95f886737..9d71db7c2 100644
--- a/wscript
+++ b/wscript
@@ -57,4 +57,4 @@ from build_file_lists import *
 from wscript_utils import *

 # Let's not depend on the locale setting of the host, set it explicitly.
-os.environ['LC_ALL'] = os.environ['LANG'] = "C"
+os.environ['LC_ALL'] = os.environ['LANG'] = "POSIX"

 # This function enables extra command line options for ./waf --help
 def options(ctx):
