diff --git a/include/clasp/core/character.h b/include/clasp/core/character.h
index 1004635ce..d1f3fc5a8 100644
--- a/include/clasp/core/character.h
+++ b/include/clasp/core/character.h
@@ -125,8 +125,8 @@ inline bool clasp_invalid_base_char_p(claspCharacter c) {
   return (c <= 32) || (c == 127);
 }

-const std::locale locale_us = std::locale("en_US.UTF-8");
-
+const std::locale locale_us = std::locale("");
+
 inline claspCharacter claspCharacter_upcase(claspCharacter code) {
   // https://en.cppreference.com/w/cpp/locale/toupper
   if (code <= 255)
